src/wallet/rpcwallet.cpp

```diff
@@ -3289,6 +3289,12 @@ UniValue generate(const JSONRPCRequest& request)
         );
     }
 
+    if (!IsDeprecatedRPCEnabled("generate")) {
+        throw JSONRPCError(RPC_METHOD_DEPRECATED, "The wallet generate rpc method is deprecated and will be fully removed in v0.19. "
```

Comment:
nit, could use same wording as https://github.com/bitcoin/bitcoin/blob/2468471e13987b1be377e1b33fe9c5cdb7a7a3e3/src/wallet/rpcwallet.cpp#L1071

Reply:
I wanted to be explicit that it was the **wallet** method that was being deprecated. Just saying `generate is deprecated` could be interpreted as the node not being able to generate blocks at all.